Goobi/pages/incMeta/Strukturdaten.jsp

```diff
@@ -204,16 +204,12 @@
 
                                 <%-- uses goobi_opac.xml --%>
 
-                                <h:selectOneMenu id="katalogauswahl" value="#{Metadaten.opacKatalog}" style="display:inline; margin-left:7px">
+                                <h:selectOneMenu id="katalogauswahl" value="#{Metadaten.opacKatalog}" style="display:inline; margin-left:7px" onchange="submit();">
```

Comment:
Please use ajax to reload the form: Remove `onchange="submit();"`, add `<a4j:support event="onchange" reRender="feldauswahl" />` between `<si:selectItems … />` and `</h:selectOneMenu>`.